dist/profile/templates/kubernetes/resources/ldap/persistentVolumeClaim-backup.yaml.erb

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: backup
+  namespace: ldap
+spec:
+  storageClassName: ""
```

Comment:
Will you add a comment on the significance of this so somebody (like me!) doesn't accidentally remove it in the future